packages/drivers/odsp-driver/src/odspDocumentStorageManager.ts

```diff
@@ -312,14 +312,7 @@ export class OdspDocumentStorageService extends OdspDocumentStorageServiceBase {
                                 method = "cache";
                             } else {
                                 method = "network";
-                                const options: ISnapshotOptions = { ...hostSnapshotOptions };
-                                // Don't fetch the blobs/deltas if it is not the first call. By default server will add
-                                // blobs and deltas to the response.
-                                if (!this.firstVersionCall) {
-                                    options.blobs = 0;
-                                    options.deltas = 0;
-                                }
-                                retrievedSnapshot = await this.fetchSnapshot(options, scenarioName);
+                                retrievedSnapshot = await this.fetchSnapshot(hostSnapshotOptions, scenarioName);
```

Comment:
There are other changes I see in the snapshot that changed this - https://github.com/microsoft/FluidFramework/pull/11784/files.
For instance, an additional param  `cacheOps` was added to `initializeFromSnapshot`. Should that be removed too?

Reply:
No, that is fine.

Reply:
It is just saying that we don't want to override previous ops fetch.